pygments breaking a test

pygments 2.14 is breaking our CI. I added a [quickfix](https://github.com/ploomber/ploomber/blob/a9d7653c20d5d638461c869ba3a3182162569d34/tests/io_mod/test_terminalwriter.py#L306) since I verified that this failure is not affecting anything critical (it's just terminal output formatting). I think this is best than pinning pygments since pining the version is worst as it'll increase the complexity of installing the package.

This is some ASCII code stuff, and I'm unsure why the pygments update is behaving in this way, here's the error:

```
===================================================================================== FAILURES ======================================================================================
________________________________________________________________ test_code_highlight[with markup and code_highlight] ________________________________________________________________

has_markup = True, code_highlight = True, expected = '{kw}assert{hl-reset} {number}0{hl-reset}\n'
color_mapping = <class 'io_mod.test_terminalwriter.color_mapping.<locals>.ColorMapping'>

    @pytest.mark.parametrize(
        ("has_markup", "code_highlight", "expected"),
        [
            pytest.param(
                True,
                True,
                "{kw}assert{hl-reset} {number}0{hl-reset}\n",
                id="with markup and code_highlight",
            ),
            pytest.param(
                True,
                False,
                "assert 0\n",
                id="with markup but no code_highlight",
            ),
            pytest.param(
                False,
                True,
                "assert 0\n",
                id="without markup but with code_highlight",
            ),
            pytest.param(
                False,
                False,
                "assert 0\n",
                id="neither markup nor code_highlight",
            ),
        ],
    )
    def test_code_highlight(has_markup, code_highlight, expected, color_mapping):
        f = io.StringIO()
        tw = terminalwriter.TerminalWriter(f)
        tw.hasmarkup = has_markup
        tw.code_highlight = code_highlight
        tw._write_source(["assert 0"], lexer='py')

>       assert f.getvalue().splitlines(keepends=True) == color_mapping.format(
            [expected])
E       AssertionError: assert ['\x1b[94mass...[39;49;00m\n'] == ['\x1b[94mass...[39;49;00m\n']
E         At index 0 diff: '\x1b[94massert\x1b[39;49;00m \x1b[94m0\x1b[39;49;00m\x1b[90m\x1b[39;49;00m\n' != '\x1b[94massert\x1b[39;49;00m \x1b[94m0\x1b[39;49;00m\n'
E         Use -v to get more diff
```

--- a/tests/io_mod/test_terminalwriter.py
+++ b/tests/io_mod/test_terminalwriter.py
@@ -38,6 +38,7 @@ def color_mapping():
             "number": "\x1b[94m",
             "str": "\x1b[33m",
             "print": "\x1b[96m",
+            "end-line": "\x1b[90m\x1b[39;49;00m",
         }
         RE_COLORS = {k: re.escape(v) for k, v in COLORS.items()}
 
@@ -301,9 +302,8 @@ class TestTerminalWriterLineWidth:
         pytest.param(
             True,
             True,
-            "{kw}assert{hl-reset} {number}0{hl-reset}\n",
+            "{kw}assert{hl-reset} {number}0{hl-reset}{end-line}\n",
             id="with markup and code_highlight",
-            marks=pytest.mark.xfail(reason="error with pygments 2.14"),
         ),
         pytest.param(
             True,

